hack/release.toml

```diff
@@ -51,6 +51,16 @@ Previously, [network device selectors](https://www.talos.dev/v1.6/talos-guides/n
         title = "KubePrism"
         description = """\
 [KubePrism](https://www.talos.dev/v1.6/kubernetes-guides/configuration/kubeprism/) is enabled by default on port 7445.
+"""
+
+    [notes.cgroupsv1]
```

Comment:
we can remove with newer pkgs

Reply:
yes, that's why I'm doing pkgs in the background :)